civitai-downloader-v2/src/api/auth.py

```
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3
+"""
+Authentication Manager for CivitAI API.
+Handles API key authentication, session management, and credential storage.
+"""
+
+import json
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)